tests/Pagination/PaginationPaginatorTest.php

```diff
@@ -174,4 +174,11 @@ public function testSimplePaginatorReturnsRelevantContextInformation()
             'prev_page_url' => '/?page=1', 'from' => 3, 'to' => 4, 'data' => ['item3', 'item4'],
         ], $p->toArray());
     }
+
+    public function testPaginatorGeneratesUrlsWithoutTrailingSlash()
```

Comment:
can you check that if you provide a trailing slash in the initial url that it's removed too